hack/update-codegen-clients.sh

```diff
@@ -27,58 +27,60 @@ BOILERPLATE_HEADER="$( pwd )/hack/boilerplate/boilerplate.go.txt"
 popd
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; go list -f '{{.Dir}}' -m k8s.io/code-generator)}
 
+# "core:v1alpha1 workload:v1alpha1 apiresource:v1alpha1 tenancy:v1alpha1 tenancy:v1beta1 apis:v1alpha1 scheduling:v1alpha1 topology:v1alpha1" \
 bash "${CODEGEN_PKG}"/generate-groups.sh "deepcopy,client" \
-  github.com/kcp-dev/kcp/pkg/client github.com/kcp-dev/kcp/pkg/apis \
-  "core:v1alpha1 workload:v1alpha1 apiresource:v1alpha1 tenancy:v1alpha1 tenancy:v1beta1 apis:v1alpha1 scheduling:v1alpha1 topology:v1alpha1" \
+  github.com/kcp-dev/edge-mc/pkg/client github.com/kcp-dev/edge-mc/pkg/apis \
+  "apis:v1alpha1" \
   --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate/boilerplate.generatego.txt \
   --output-base "${SCRIPT_ROOT}" \
-  --trim-path-prefix github.com/kcp-dev/kcp
+  --trim-path-prefix github.com/kcp-dev/edge-mc
 
 pushd ./pkg/apis
 ${CODE_GENERATOR} \
-  "client:outputPackagePath=github.com/kcp-dev/kcp/pkg/client,apiPackagePath=github.com/kcp-dev/kcp/pkg/apis,singleClusterClientPackagePath=github.com/kcp-dev/kcp/pkg/client/clientset/versioned,headerFile=${BOILERPLATE_HEADER}" \
-  "lister:apiPackagePath=github.com/kcp-dev/kcp/pkg/apis,headerFile=${BOILERPLATE_HEADER}" \
-  "informer:outputPackagePath=github.com/kcp-dev/kcp/pkg/client,singleClusterClientPackagePath=github.com/kcp-dev/kcp/pkg/client/clientset/versioned,apiPackagePath=github.com/kcp-dev/kcp/pkg/apis,headerFile=${BOILERPLATE_HEADER}" \
+  "client:outputPackagePath=github.com/kcp-dev/edge-mc/pkg/client,apiPackagePath=github.com/kcp-dev/edge-mc/pkg/apis,singleClusterClientPackagePath=github.com/kcp-dev/edge-mc/pkg/client/clientset/versioned,headerFile=${BOILERPLATE_HEADER}" \
+  "lister:apiPackagePath=github.com/kcp-dev/edge-mc/pkg/apis,headerFile=${BOILERPLATE_HEADER}" \
+  "informer:outputPackagePath=github.com/kcp-dev/edge-mc/pkg/client,apiPackagePath=github.com/kcp-dev/edge-mc/pkg/apis,singleClusterClientPackagePath=github.com/kcp-dev/edge-mc/pkg/client/clientset/versioned,headerFile=${BOILERPLATE_HEADER}" \
```

Comment:
Perhaps the way github displays review comments obscuring my meaning.  There are two lines that you have been thrashing, new line number 40 and 42 in the current revision.  My complaint each time has been about just one of the two lines, and you changed both.  The thing I have been trying to say is that if both lines have their content ordered in the same way as the _corresponding original_ then the PR is easier to review.  This has always required changing just one of the two lines.

Reply:
To be clear: in the current revision of this PR, new line 40 corresponds with old line 39 and the two have their clauses in the same order --- making it easy to review the change to this line.  In contrast, new line 42 corresponds with old oline 41 and the two have their clauses in different orders, making the PR harder to review.  OTOH, old lines 39 and 41 are inconsistent in their ordering of their clauses --- so perhaps you are intentionally making one ordering change.

Reply:
here is the current state of the PR: (separated the lines at commas with a CR for clarity)

line 40:  

"client:outputPackagePath=github.com/kcp-dev/edge-mc/pkg/client,
apiPackagePath=github.com/kcp-dev/edge-mc/pkg/apis,
singleClusterClientPackagePath=github.com/kcp-dev/edge-mc/pkg/client/clientset/versioned,
headerFile=${BOILERPLATE_HEADER}" \

line 42:  
"informer:outputPackagePath=github.com/kcp-dev/edge-mc/pkg/client,
apiPackagePath=github.com/kcp-dev/edge-mc/pkg/apis,
singleClusterClientPackagePath=github.com/kcp-dev/edge-mc/pkg/client/clientset/versioned,
headerFile=${BOILERPLATE_HEADER}" \

the two lines are identical with the exception of the key 'client' and 'informer'

can you tell me how these lines are mis-ordered?

Reply:
Right.  Now look equally carefully at the corresponding two lines as they exist in `main`.  You will see that they are not consistent with each other.  My original review remark was that since your PR is changing the ordering in _one_ of the two lines, it makes the PR harder to review regarding that line.  My second remark was asking whether you were saying you think it is more important to have the two lines consistent with each other than consistent with the corresponding lines in kcp-dev/kcp (where the inconsistency originates).